src/test/java/uk/gov/pay/ledger/transaction/service/TransactionServiceTest.java

```diff
@@ -43,8 +39,8 @@
     private TransactionSearchParams searchParams;
 
     @Before
-    public void setUp() throws MalformedURLException {
-        transactionService = new TransactionService(mockTransactionDao);
+    public void setUp() {
+        transactionService = new TransactionService(mockTransactionDao, new TransactionEntityFactory(new ObjectMapper()));
```

Comment:
I don't think we use/rely on any of the dropwizard customisations but I *think* it's recommended to use `Jackson.newObjectMapper()` which builds an object mapper with some extra plugins, or at least that's a more realistic simulation of what would get provided through DI

Reply:
I see. Thanks!